### DISCUSSION TITLE
Recipe Buddy

### DISCUSSION THREAD

**Discussion by TudorAndrei** [coollabsio/coolify#4166] [Service Template Requests]
It would be great to launch this along with Grocy

https://github.com/georgegebbett/recipe-buddy

**Comment by lloydrichards** [comment]
maybe you're interested, but I just added #4262 as an alternative 🤷‍♂️

**Reply by TudorAndrei** [reply]
looks, cool, but recipe-buddy integrates well with gorcy which already has meal plans
